### PR TITLE
Add Cookie Policy, disclaimer and cookie banner

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Add GTM plugin based on (https://github.com/LukasGentele/docusaurus-gtm-plugin/blob/master/index.js and https://github.com/facebook/docusaurus/blob/main/packages/docusaurus-plugin-google-gtag/src/index.js)
  -  I tested both of them and also the @docusaurus/plugin-google-analytics and @docusaurus/plugin-google-gtag, but for them the URL is different. It's still valid based on `Tag Assistant` but in docs:
  - `https://www.googletagmanager.com/gtag/js?id=${trackingID}`:
     > (..) that allows you to send event data to Google Analytics, Google Ads, and Google Marketing Platform" 
     
      _see: https://developers.google.com/analytics/devguides/collection/gtagjs_
      I don't know if that will properly dump data on the server side for Google Tag Manger
  - `https://www.googletagmanager.com/gtm.js?id=`:
     > " It enables Tag Manager to fire tags by inserting gtm.js into the page (or through the use of an iframe when JavaScript isn't available)."
     
     _see https://developers.google.com/tag-manager/quickstart/_
     Below you can see page build from PR sources:
     Screenshot 2021-09-01 at 13 32 50

- Add cookie banner base on https://github.com/buildo/react-cookie-banner
- Add Disclaimer and Cookie Policy generated by Termly. I tested few generators and this is the best as it's free and generates readable docs. Other solutions that I found were paid or quite dummy, as https://www.cookiepolicygenerator.com/

## Deployed site

https://f7c1ba35.website-8qa.pages.dev/

After changes:
https://8bde7800.website-8qa.pages.dev/

## Related issue(s)

Fixes https://github.com/capactio/website/issues/41

### CODE REVIEW

File: src/components/CookiesBanner.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+	if (!init || !cookies || cookies.get("accepts-cookies")) {
+		return null;
+	}
+
+	return (
+		<CookieBanner
+			disableStyle={true}
+			className={styles.banner}
+			message="By continuing to use our site you are consenting to Capact using cookies in accordance with our "
```
Comment (mszostok) [inline src/components/CookiesBanner.tsx:22]:
```suggestion
			message="By continuing to use our site, you consent to Capact using cookies in accordance with our "
```

File: src/components/CookiesBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, {FunctionComponent, useEffect, useState} from "react";
+import {Cookies} from "react-cookie";
+import CookieBanner from 'react-cookie-banner';
```
Comment (pkosiec) [inline src/components/CookiesBanner.tsx:3]:
```suggestion
import CookieBanner, {Cookies} from 'react-cookie-banner';
```

File: src/components/CookiesBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+export const CookiesMessageBanner: FunctionComponent = () => {
+	const cookies = new Cookies();
+	const [init, setInit] = useState<boolean>(false);
+
+	useEffect(() => {
+		setInit(true);
+	}, []);
+
+	if (!init || !cookies || cookies.get("accepts-cookies")) {
```
Comment (pkosiec) [inline src/components/CookiesBanner.tsx:14]: Looks like the lines are not needed - I checked both in dev and prod build:
```suggestion
	if (!cookies || cookies.get("accepts-cookies")) {
```

**Review by pkosiec — COMMENTED** [review]
Looks good 👌 Few comments.

File: src/components/CookiesBanner.module.css [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    text-align: center;
+}
+
+.banner button:global(.button-close) {
+    border: 1px solid var(--capact-black);
+    border-radius: var(--ifm-button-border-radius);
+    background: transparent;
+    color: var(--capact-black);
+    cursor: pointer;
+
```
Comment (pkosiec) [inline src/components/CookiesBanner.module.css:26]:
```suggestion
```

File: src/components/CookiesBanner.module.css [hunk 5]
```diff
@@ -0,0 +1,62 @@
+.banner {
+    position: fixed;
+    width: 100%;
+    z-index: 9999;
+    bottom: 0;
+    padding: var(--ifm-navbar-padding-vertical) var(--ifm-navbar-padding-horizontal);
+    background: var(--capact-gray-lighter);
+    border-top: 1px solid var(--capact-gray-light);
+
```
Comment (pkosiec) [inline src/components/CookiesBanner.module.css:9]:
```suggestion
```

**Review by pkosiec — APPROVED** [review]

**Pull request merged**